File: javascript/selection_sort/src/selection_sort.js
const sort = (arr) => {
    for ( let i = 0; i < arr.length - 1; i++ ) {
        for ( let j = i + 1; j < arr.length; j++ ) {
            if ( arr[j] < arr[i] ) {
                let temp = arr[j];
                arr[j] = arr[i];
                arr[i] = temp;
            }
        }
    }
    return arr;
}

exports.sort = sort;

File: python/selection_sort/selection_sort.py
def sort(lst):
    for i in range(len(lst) - 1):
        for j in range(i + 1, len(lst)):
            if lst[j] < lst[i]:
                temp = lst[j]
                lst[j] = lst[i]
                lst[i] = temp
    return lst
